specification/automation/resource-manager/Microsoft.Automation/preview/2023-05-15-preview/deletedAutomationAccount.json

```diff
@@ -0,0 +1,144 @@
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Automation/deletedAutomationAccounts": {
+      "get": {
+        "tags": [
+          "DeletedAutomationAccounts"
+        ],
+        "operationId": "deletedAutomationAccounts_ListBySubscription",
+        "description": "Retrieve deleted automation account.",
+        "x-ms-examples": {
+          "Get deleted automation account": {
+            "$ref": "./examples/getDeletedAutomationAccount.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DeletedAutomationAccountListResult"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../common/v1/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DeletedAutomationAccountListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DeletedAutomationAccount"
+          },
+          "description": "Gets or sets the list of deleted automation accounts."
+        }
+      },
+      "description": "The response model for the list deleted automation account."
+    },
+    "DeletedAutomationAccount": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DeletedAutomationAccountProperties",
+          "x-ms-client-flatten": true,
+          "description": "Gets or sets the automation account properties."
+        },
+        "id": {
+          "description": "The resource id.",
+          "type": "string"
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets name of the resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "The resource type."
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets the location of the resource."
+        }
+      },
+      "description": "Definition of the deleted automation account type."
+    },
+    "DeletedAutomationAccountProperties": {
+      "type": "object",
+      "properties": {
+        "automationAccountResourceId": {
```

Comment:
>automationAccountResourceId [](http://example.com/codeflow?start=9&length=27)

Is it an azure resource ID? If so, please add "format": "arm-id" and "x-ms-arm-id-details".

[https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details)

Reply:
We have not modified this resource. It came as part of the last stable version.